client/src/templates/Challenges/fill-in-the-blank/show.tsx

```diff
@@ -180,8 +180,8 @@ class ShowFillInTheBlank extends Component<
 
     const blankAnswers = blanks.map(b => b.answer);
 
-    const newAnswersCorrect = userAnswers.map(
-      (userAnswer, i) => userAnswer === blankAnswers[i]
+    const newAnswersCorrect = userAnswers.map((userAnswer, i) =>
+      userAnswer && userAnswer.trim() === blankAnswers[i]
```

Comment:
```suggestion
      !!userAnswer && userAnswer.trim() === blankAnswers[i]
```

Easy fix - we just boolean it.

Reply:
Okay done.